js/app.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const requestInfoPerson4 = new XMLHttpRequest();
+
+function createRequestInfoPerson4() {
+  const person = JSON.parse(this.responseText);
+  const person4Name = document.querySelector('#person4Name');
+  person4Name.innerHTML = person.name;
+
+  const requestHome = new XMLHttpRequest();
+
+  function requestET4Home() {
+    const home = JSON.parse(this.responseText);
+    const requestHome = document.querySelector('#person4HomeWorld');
+    requestHome.innerHTML = home.name;
+  }
+
+  requestHome.addEventListener('load', requestET4Home);
+  requestHome.open('GET', person.homeworld);
+  requestHome.send();
+}
+requestInfoPerson4.addEventListener('load', createRequestInfoPerson4);
+requestInfoPerson4.open('GET', 'https://swapi.co/api/people/4/');
+requestInfoPerson4.send();
+
+/*
+      Person14 request
+*/
+
+const requestInfoPerson14 = new XMLHttpRequest();
+
+function createRequestInfoPerson14() {
+  const person = JSON.parse(this.responseText);
+  const person14Name = document.querySelector('#person14Name');
+  person14Name.innerHTML = person.name;
+
+  const requestSpecies = new XMLHttpRequest();
+
+  function request14Species() {
+    const species = JSON.parse(this.responseText);
+    const requestSpecies = document.querySelector('#person14Species');
+    requestSpecies.innerHTML = species.name;
+  }
```

Comment:
Nothing technically wrong with this but probably better to get in the habit of not declaring functions in another function or for loop. Makes code hard to read/refactor. No reason you can't define the function outside the function and invoke it with any necessary parameters inside the function